recipes/tsne/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "tsne" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "c523c429da31b251d20388d03f75ea8ef24d066b38e123fe44467bd5d26bb9f5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/danielfrg/tsne/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py3k and py<37]
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - cython
+    - numpy
+    - openblas
```

Comment:
See https://conda-forge.org/docs/maintainer/knowledge_base.html#blas

Reply:
I think I have addressed the blas issue. Is this now ready to be merged?